modules/glog/metadata.json

```diff
@@ -1,17 +1,18 @@
 {
-  "homepage": "https://github.com/google/glog",
-  "maintainers": [
-    {
-      "email": "bcr-maintainers@bazel.build",
-      "name": "No Maintainer Specified"
-    }
-  ],
-  "repository": [
-    "github:google/glog"
-  ],
-  "versions": [
-    "0.5.0",
-    "0.6.0"
-  ],
-  "yanked_versions": {}
+    "homepage": "https://github.com/google/glog",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
Do you mind adding yourself as maintainer here?

Reply:
Added me as a maintainer